### PR TITLE
9224 comma separation

### PR DESCRIPTION
**Pull request by grimmoc**
## Description
Tags containing commas are now split up into separate tags on client side to match server side processing behaviour.

## Related Issue

- implements https://github.com/owncloud/web/issues/9224

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [x] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

File: packages/web-app-files/src/components/SideBar/TagsPanel.vue [hunk 1]
```diff
@@ -181,7 +182,7 @@ export default defineComponent({
       createOption,
       isOptionSelectable,
       showSelectNewLabel,
-      save
+      save,
```
Comment (kulmann) [inline packages/web-app-files/src/components/SideBar/TagsPanel.vue:185]: Please always run `pnpm lint --fix` on save in your IDE or before you commit. ;-) This comma fails in the linting step.

File: packages/web-app-files/src/components/SideBar/TagsPanel.vue [hunk 2]
```diff
@@ -11,6 +11,7 @@
         :options="availableTags"
         taggable
         push-tags
+        :select-on-key-codes="[188, 13]"
```
Comment (kulmann) [inline packages/web-app-files/src/components/SideBar/TagsPanel.vue:14]: No magic numbers please :-)

Comment (kulmann) [reply packages/web-app-files/src/components/SideBar/TagsPanel.vue:14]: We already have the dependency `keycode` (see root level package.json). You can use that to create an array with human readable keys in the setup function, return that from the setup function, and use it here in the template. (e.g. `keycode('enter')` returns `13`, you can just throw that into an array.

File: packages/web-app-files/src/components/SideBar/TagsPanel.vue [hunk 3]
```diff
@@ -74,6 +76,7 @@ type TagOption = {
 
 export default defineComponent({
   name: 'TagsPanel',
+  methods: { keycode },
```
Comment (kulmann) [inline packages/web-app-files/src/components/SideBar/TagsPanel.vue:79]: Please don't introduce new options api code. You can return the keycode function in the setup function.

Comment (grimmoc) [reply packages/web-app-files/src/components/SideBar/TagsPanel.vue]: hmm, I should better not rely on method importing using WebStorm or at least I should double check before committing :/

**Review by JammingBen — APPROVED** [review]
👍

**Pull request merged**